core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitorSpec.scala

```diff
@@ -264,56 +275,65 @@ class AvroUpsertMonitorSpec(_system: ActorSystem)
     makeOpsJsonString(opsList)
   }
 
-  List(1, 2, 20, 250, 2345, 12345) foreach { upsertQuantity =>
-    it should s"upsert $upsertQuantity entities" in withTestDataApiServices { services =>
-      val timeout = 120000 milliseconds
-      val interval = 500 milliseconds
-      val importId1 = UUID.randomUUID()
-
-      // add the imports and their statuses to the mock importserviceDAO
-      val mockImportServiceDAO = setUp(services)
-      mockImportServiceDAO.imports += (importId1 -> ImportStatuses.ReadyForUpsert)
-
-      // create upsert json file
-      val contents = makeOpsJsonString(upsertQuantity)
+  List("Import Service", "cWDS") foreach { origin =>
```

Comment:
we run these happy-path tests for both cWDS and Import Service code paths. I was almost tempted to run ALL tests in this file for both code paths, but that feels like it adds a lot of time for little benefit; since Import Service is on its way out I'm ok with it